base/templates/base/base.html

```diff
@@ -1,129 +1,131 @@
+{% load static %}
+{% load bootstrap3 %}
```

Comment:
- I always put the tags for loading of static files, boostrap etc. on top of template file (for having a better overview), that's why I factored it out of the head.
- I also used the more common {% load static %} instead of {% load staticfiles %}, because we don't host static files on external services - right?: https://stackoverflow.com/questions/24238496/what-is-the-difference-between-load-staticfiles-and-load-static